### DISCUSSION TITLE
Feature Requests.

### DISCUSSION THREAD

**Discussion by SuhasDissa** [you-apps/ConnectYou#297] [Ideas]
Hi everyone,

We're starting this thread to help keep the issue log clean and focused on the most critical issues. Please use this thread to post your feature requests and design suggestions.

We appreciate your feedback!

**Comment by Turhvjbufv** [comment]
Add an option that when you send a message it will wait a custom amount of seconds and then send it, and you will have two buttons next to the message that you are sending, one is to disable sending the message and one is to send immediately, QKsms helped me with the idea(since they have something very similar), maybe It will help you with writing the code

**Comment by xusiai** [comment]
I would appreciate a pure black / amoled theme option. Lineage ships a black color option in their MD3 implementation but for those of us stuck with Google - who does not include black - adding a toggle to use a black background with the MD3 accent would be an appreciated addition.

**Comment by Mikapo3** [comment]
I would like a feature that allows you to choose when the app gets locked behind biometric authentication. For example you could choose to have the app locked after 2 minutes of not using it.
